src/tsx/liveScreen.tsx

```diff
@@ -168,26 +180,39 @@ class LiveScreen extends React.PureComponent<IProps, IState> {
   private handleKey(event: KeyboardEvent) {
     // Only accept keyboard shortcuts on first press and when nothing is focused
     if (document.activeElement === document.body && !event.repeat) {
-      if (event.key === "s") {
+      let keyCombination = "";
+      if (event.altKey) keyCombination += "alt+";
+      if (event.ctrlKey) keyCombination += "ctrl+";
+      if (event.shiftKey) keyCombination += "shift+";
+      keyCombination += event.key;
+
+      if (keyCombination === "s") {
```

Comment:
Maybe later, but should we replace the "a" and "s" shortcuts with buttons to free up keys for custom shortcuts? They are not that common or time-critical.

Reply:
True. Will add an issue.